Source/FakeItEasy.Specs/AssertingOnSetOnlyProperties.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace FakeItEasy.Specs
+{
+    using System.Diagnostics.CodeAnalysis;
+    using Xbehave;
+
+    public  class AssertingOnSetOnlyProperties
+    {
+        [Scenario]
+        public void SetOnlyProperties(
+            ISetOnly setOnly)
+        {
+            "establish"
+                .x(() => setOnly = A.Fake<ISetOnly>());
+
+            "when assertion on set only properties"

```

Comment:
Oh, and "asserting". Maybe this will all be covered in the FakeItEasy-team-lead step name rewriting?